ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -22,6 +23,9 @@
    [compojure.core :refer [DELETE GET POST PUT context routes]]
    [drift.execute :as drift]))
 
+(def URS_TOKEN_MAX_LENGTH 100)
+(def CMR_INGEST_SEPARATOR "CMR_INGEST:")
```

Comment:
I think this is supposed to signify we need to check whether someone should be allowed to write to NASA systems (broader than just ingest access). So I think the name should be something like WRITE_ACCESS rather than CMR_INGEST.

Reply:
Also note we'll need to do this in other applications, so `common-app-lib` is the place to add this code rather than `ingest.api.routes`.

Reply:
Renaming done. The move to common-app-lib will be done later in real implementation (not prototyping).